src/core/Select/Select.test.tsx

```diff
@@ -336,3 +336,12 @@ it('should use custom renderer for menu items', () => {
   expect(menuItems[1].style.color).toEqual('green');
   expect(menuItems[2].style.color).toEqual('red');
 });
+
+it('should render small and large sizes', () => {
```

Comment:
Jest supports iterating trough test params with `it.each`, you can look at line 191 in this file.

Reply:
Oh that's cool